dev/com.ibm.ws.webcontainer.cors/bnd.bnd

```diff
@@ -44,6 +44,7 @@ instrument.classesExcludes: com/ibm/ws/webcontainer/cors/internal/resources/*.cl
 
 -buildpath: \
 	com.ibm.websphere.javaee.servlet.3.0;version=latest,\
+	io.openliberty.webcontainer.servlet.6.0.internal,\
```

Comment:
Did you mean to add this? What is it for?

Reply:
This was something we added during debugging, I've since removed it.